### DISCUSSION TITLE
Help with quick disconnect shortcut and weird media key shortcut

### DISCUSSION THREAD

**Discussion by landozz** [nefarius/DsHidMini#193] [Support]
Hey Ive been using dshidmini for a while for emulating playstation games and I usually use the l1+r1+ps button for quick disconnect of the controller instead of holding the ps button for a few seconds, but Ive noticed that whenever I press l1+ps button music or whatever last active window of a video starts playing, basically a play/stop media key and it does this every time I turn it off with the shortcut. Im not sure how to disable it since theres no options for it in the app or bthps3, so can anyone help turn this off? not entirely sure what program caused this either and it doesnt happen with any other controllers I have, but any help would be great.

**Comment by Kanuan** [comment]
1. In which HID Device Mode is the controller in?
    - You can check this by opening the DSHMC executable
2. Do you use Steam? Is it running in the background when this behavior occurs?

**Reply by landozz** [reply]
>     2. Do you use Steam? Is it running in the background when this behavior occurs?
 it was steam after all, I was able to change it and now I can disconnect without bothering my music, thanks!
